### PR TITLE
Document transactions support in Hive connector

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Review by aczajkowski — APPROVED** [review]
LGTM % Just thinking if we shouldn't put on some common connector page list (one item length now 😄  ) of connectors which consider transactions at all. It would be straight forward and easy to find.

**Comment by hashhar** [comment]
> LGTM % Just thinking if we shouldn't put on some common connector page list (one item length now 😄 ) of connectors which consider transactions at all. It would be straight forward and easy to find.

It's generally difficult to keep information up to date in summarized pages. It's easier to update/verify docs for a connector are true.

If we want to go that route we'd want it to be "generated" IMO.

File: docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -666,6 +666,7 @@ features:
   * :ref:`sql-views-management`
 
 * :ref:`sql-security-operations`, see also :ref:`hive-sql-standard-based-authorization`
+* :ref:`sql-transactions`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/hive.rst:669]: cc: @losipiuk confirm if transactions can be used with all types of Hive tables?

**Comment by mosabua** [comment]
> > LGTM % Just thinking if we shouldn't put on some common connector page list (one item length now 😄 ) of connectors which consider transactions at all. It would be straight forward and easy to find.
> 
> It's generally difficult to keep information up to date in summarized pages. It's easier to update/verify docs for a connector are true.
> 
> If we want to go that route we'd want it to be "generated" IMO.

Agreed .. generating stuff is painful though. In most cases users are not really interested in a feature like transactions support in general across all connectors.. they just want to know what "their" connector supports. Maintaining lists and such is hard work .. just look at the list of functions we have...

**Comment by mosabua** [comment]
Thank you for the merge @findepi .. and the review and input @hashhar and @aczajkowski

**Pull request merged**